extension/content.js

```diff
@@ -344,6 +344,18 @@ function addFilterCommentsByYou() {
 	}
 }
 
+function addProjectNewLink() {
+	const projectNewLink = `<a href="/${repoUrl}/projects/new" class="btn btn-sm" id="refined-github-project-new-link">Add a project</a>`;
+	$('#projects-feature').next('p.note').after(projectNewLink);
+}
+
+function removeProjectsTab() {
+	const projectsTab = $('.js-repo-nav').find('.reponav-item[data-selected-links^="repo_projects"]');
+	if (projectsTab.find('.Counter').text() === '0') {
```

Comment:
We should check that the `projectsTab` even exists. If not the `find()` is going to throw an error.